### DISCUSSION TITLE
v0.5.3 Pre-release thread

### DISCUSSION THREAD

**Discussion by masenf** [reflex-dev/reflex#3414] [Release]
# Try out the Prerelease

Please report any problems you encounter in this discussion thread or as separate issues on reflex-dev/reflex.

```console
pip install 'reflex~=0.5.3a'
```

# Schedule

2024-05-31 17:45 PDT: 0.5.3a1 pre-release published for testing
2024-06-03: Planned Release of 0.5.3
2024-06-05 13:27 PDT: Public release of 0.5.3

# Release Notes

## New Features

* External assets by @abulvenz in https://github.com/reflex-dev/reflex/pull/3220
* add mapping between client_token and socket id by @Lendemor in https://github.com/reflex-dev/reflex/pull/3388
* add config knob for react_strict_mode by @Lendemor in https://github.com/reflex-dev/reflex/pull/3389
* [REF-2202] Implement event handlers for Plotly by @masenf in https://github.com/reflex-dev/reflex/pull/3397

## Improvements
* Radix Themes + Tailwind Harmony by @masenf in https://github.com/reflex-dev/reflex/pull/3355
* Explicit deps and interval for computed vars by @benedikt-bartscher in https://github.com/reflex-dev/reflex/pull/3231
* Add domain prop for the PolarRadiusAxis component by @TG199 in https://github.com/reflex-dev/reflex/pull/3349
* [REF-2710]Recommend Running with `REFLEX_USE_NPM=1` if npm run fails after installing packages with bun. by @ElijahAhianyo in https://github.com/reflex-dev/reflex/pull/3399
* refactor: radix section default size by @Jishnu-Nandhiath in https://github.com/reflex-dev/reflex/pull/3406
* [REF-2978] Ignore Redis config_set for AWS ElastiCache by @masenf in https://github.com/reflex-dev/reflex/pull/3401
* [REF-1356] Track changes applied to `Base` subclass via helper method. by @masenf in https://github.com/reflex-dev/reflex/pull/2242
* include resize and radius props to the text area component by @carlosabadia in https://github.com/reflex-dev/reflex/pull/3383

### Each component import is now independently lazy

Save time by avoiding imports for components which are not used in the app.

* [REF-2821]Improve Dynamic Imports by @ElijahAhianyo in https://github.com/reflex-dev/reflex/pull/3345

## Bug Fixes

* Layout property not pushed through on rx.plotly by @HongyuHansonYao in https://github.com/reflex-dev/reflex/pull/3394

## Other Changes

* [REF-2895]Benchmarks getting skipped on merge by @ElijahAhianyo in https://github.com/reflex-dev/reflex/pull/3369
* Updated in/readme.md with current readme  by @sagarhedaoo in https://github.com/reflex-dev/reflex/pull/3333
* update image for readme by @tgberkeley in https://github.com/reflex-dev/reflex/pull/3395

## New Contributors
* @sagarhedaoo made their first contribution in https://github.com/reflex-dev/reflex/pull/3333
* @TG199 made their first contribution in https://github.com/reflex-dev/reflex/pull/3349
* @HongyuHansonYao made their first contribution in https://github.com/reflex-dev/reflex/pull/3394
* @carlosabadia made their first contribution in https://github.com/reflex-dev/reflex/pull/3383

**Full Changelog**: https://github.com/reflex-dev/reflex/compare/v0.5.2...reflex-0.5.3

**Comment by sebastianjanisch** [comment]
There seems to be an issue with custom stylesheets in this pre-release:

![image](https://github.com/reflex-dev/reflex/assets/162770489/c5aa835b-dafb-4345-a781-d23ebe2090e2)
![image](https://github.com/reflex-dev/reflex/assets/162770489/5597995d-ff22-4ef5-aca4-71add40ac598)

**Comment by masenf** [comment]
0.5.3 is released https://github.com/reflex-dev/reflex/releases/tag/v0.5.3
